tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -363,9 +361,9 @@ string DebugString(const nvinfer1::Permutation& permutation, int len) {
 
 string DebugString(const nvinfer1::ITensor& tensor) {
   return StrCat("nvinfer1::ITensor(@", reinterpret_cast<uintptr_t>(&tensor),
-                ", name=", tensor.getName(),
-                ", dtype=", DebugString(tensor.getType()),
-                ", dims=", DebugString(tensor.getDimensions()), ")");
+                ", name=", tensor.getName(), ", dtype=",
```

Comment:
Can you revert changes like this in this PR? They contributed to a big part of the diff in this and other files.